todos-mongokitten-openapi/Sources/todos-mongokitten-openapi/API.swift

```diff
@@ -0,0 +1,77 @@
+@preconcurrency import MongoKitten
+import OpenAPIRuntime
+import TodosOpenAPI
+
+struct TodoModel: Codable {
+    let _id: ObjectId
+    let items: [String]
```

Comment:
What is `items`? The todo doesn't seem to have a title, or status. Is this somehow store in `items`